### PR TITLE
Paradise Blue™️ 

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Adds Paradise Blue, a brand new way to show how important you are and how your opinions matter in space station 13! If you're a [Patron](https://www.patreon.com/ParadiseStation) or a [BYOND member](https://www.byond.com/membership), you'll get a blue checkmark next to every message of yours.

## Why It's Good For The Game
People who paid money deserve something!
(also you should really consider becoming a donor for Paradise and/or support BYOND, both parties would very much appreciate it and it keeps the game you love alive).

## Images of changes
image

## Testing
yes

### CODE REVIEW

**Comment by lewcc** [comment]
I'm still dumbfounded that this works, that you can just. Put icons in maptext.

File: code/datums/chatmessage.dm [hunk 1]
```diff
@@ -132,7 +152,12 @@
 			symbol = "<span style='font-size: 9px; color: #3399FF;'>*</span> "
 			size = size || "small"
 		else
-			symbol = null
+			// afd or not, we don't want to actually make people's lives harder
+			var/no_symbol_regardless = (owned_by.holder?.fakekey || (owned_by.prefs.toggles2 & PREFTOGGLE_2_ANON) || !owned_by.show_checkmark)
+			if((owned_by.IsByondMember() || owned_by.donator_level) && !no_symbol_regardless)
```
Comment (AffectedArc07) [inline code/datums/chatmessage.dm:157]: Please do this check on `client/New()`.

iirc `IsByondMember()` does an external lookup and we dont need extra overhead on every single runechat.

Comment (Contrabang) [reply code/datums/chatmessage.dm:157]: Does that mean every OOC chat also checks isbyondmember? Or maybe we already have a cache somewhere

Comment (AffectedArc07) [reply code/datums/chatmessage.dm:157]: Yeah we cache it as `prefs.unlock_content`

Comment (lewcc) [reply code/datums/chatmessage.dm:157]: good point, I think donator level shows it regardless.

Comment (lewcc) [reply code/datums/chatmessage.dm]: Should be fixed

**Review by farie82 — APPROVED** [review]

**Pull request merged**